.github/workflows/dotnet-ci.yml

```
@@ -27,3 +29,16 @@ jobs:
       - name: Build
         working-directory: ./code/
         run: dotnet build
+  test:
+    needs: build
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
```

actionlint (1.7.10)

[error] the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)